src/app/projects/project-list/project-list.component.ts

```diff
@@ -351,6 +351,12 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     return overlappingDistrictsListString;
   }
 
+	stringifyProjectTypes(projectTypes) {
```

Comment:
You could also chain these array methods, just like jquery used to do!

```suggestion
	stringifyProjectTypes(projectTypes: ProjectType[]) {
	    // Filter by selected types, then get the string of their name, then join as a single string.
	    return projectTypes.filter(pt => true === pt.checked).map(pt => pt.name).join(', ') || '';
	    }
```

Reply:
ok I'll try that approach next time